### DISCUSSION TITLE
stillImageSource - can anyone confirm this is working?

### DISCUSSION THREAD

**Discussion by cchewanik** [Sunoo/homebridge-camera-ffmpeg#1452] [General]
Hi, I am trying to get a stillImageSource from a detection from Frigate instead of the camera, but it seems it is ignoring the URL I have entered, and is using a frame from the feed instead. I have confirmed my snapshot URL is working (it is obvious as it has bounding boxes...).  Can anyone confirm stillImageSource is working for them?

Thanks

Excerpt of my config...

{
                    "disable": false,
                    "name": "west gate motion",
                    "motion": true,
                    "motionTimeout": 15,
                    "unbridge": true,
                    "hsv": false,
                    "prebuffering": true,
                    "prebufferLength": 4,
                    "videoConfig": {
                        "source": "-re -rtsp_transport tcp -i rtsp://admin:xxx@x:554/h264Preview_01_main",
                        "subSource": "-re -rtsp_transport tcp -i rtsp://admin:xxx@x:554/h264Preview_01_main",
                        "stillImageSource": "-i http://x:8001/api/west_gate/any/snapshot.jpg",
                        "maxStreams": 2,
                        "maxWidth": 0,
                        "maxHeight": 0,
                        "maxFPS": 0,
                        "maxBitrate": 0,
                        "vcodec": "copy",
                        "acodec": "libfdk_aac",
                        "encoderOptions": "-bf 0",
                        "audio": true
                    },
                    "mqtt": {
                        "motionTopic": "nodered/west_motion",
                        "motionMessage": "true",
                        "motionResetTopic": "nodered/west_motion",
                        "motionResetMessage": "false"
                    },
                    "smtp": {
                        "email": "west gate"
                    },
                    "videoanalysis": {
                        "active": false
                    }
                },

**Comment by Sunoo** [comment]
That doesn’t look like a config for this plugin.
